docs/adapters/mikro-orm.md

```diff
@@ -0,0 +1,131 @@
+---
+id: mikro-orm
+title: MikroORM
+---
+
+To use this Adapter, you need to install Mikro ORM, the driver that suits your database, and the separate `@next-auth/mikro-orm-adapter` package:
+
+:::warning
+Please make sure to use the `next` tagged version of mikro-orm. Refer [here](https://github.com/jonahallibone/mikro-orm-nextjs) for a setup example with NextJS.
+:::
+
+:::warning
+Please make sure to use the `next` tagged version of the adapter.
+:::
+
+```bash npm2yarn
+npm install next-auth@beta @next-auth/mikro-orm-adapter@next @mikro-orm/core@next @mikro-orm/[YOUR DRIVER]@next
```

Comment:
```suggestion
npm install next-auth @next-auth/mikro-orm-adapter @mikro-orm/core@next @mikro-orm/[YOUR DRIVER]@next
```

Actually, just drop the `next`, `beta`! Next stop is at `latest` :steam_locomotive: